.github/workflows/flash_analysis.yml

```diff
@@ -92,25 +129,32 @@ jobs:
           echo "timestamp=$(date +'%Y-%m-%dT%H:%M:%S')" >> $GITHUB_OUTPUT
 
       - name: Create or update comment
+        # This can't be moved to the job-level conditions, as GH actions don't allow a job-level if condition to access the env.
```

Comment:
An alternative to using `env` is to introduce a new step that sets the outputs, but it should be fine as is right now.